arkouda/pdarrayclass.py

```diff
@@ -1168,6 +1188,38 @@ def info(pda : Union[pdarray, str]) -> str:
         raise TypeError("info: must be pdarray or string".format(pda))
         return generic_msg(cmd="info", args="{}".format(pda))
 
+def list_registry() -> list[str]:
```

Comment:
I think the type string here should be List[str] vs list[str]
The python typing module uses the capitalized for for annotating typing info.

Reply:
@glitch yup, but I believe we can do list[str] (better option, IMHO) when we officially upgrade to 3.8 as minimum Python version (pretty sure it's 3.8, will test to confirm).